src/main/java/com/sap/sgs/phosphor/fosstars/model/value/Vulnerability.java

```diff
@@ -50,11 +51,13 @@
   /**
    * A date when the vulnerability was introduced.
    */
+  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd", locale = "us_US")
```

Comment:
why not pattern yyyy-MM-dd'T'HH:mm:ss.SSSZ here?

Reply:
I think that `yyyy-MM-dd` is enough for a vulnerability because it is not necessary to know exact time when it was introduced or fixed. Then, `yyyy-MM-dd` is easier to read.

Reply:
Agree